lib/unit-ruby/deposit_account.rb

```diff
@@ -19,6 +19,7 @@ class DepositAccount < APIResource
     attribute :close_reason, Types::String, readonly: true # Optional. The reason the account was closed, either ByCustomer or Fraud.
 
     belongs_to :customer, class_name: 'Unit::IndividualCustomer'
+    has_many :customers, class_name: 'Unit::IndividualCustomer'
```

Comment:
Might be worth calling out in a comment that this is for joint accounts

Reply:
@ianyamey thanks for commenting on this - I'd actually appreciate your thoughts on this specific instance. I thought it would be strange for the user of ruby gem to have to know whether the particular resource belongs to `customer` or to `customers`, so in my implementation of adding `customers` enabled it to also work when there was only one particular customer added to the account. 

ie if `.customers` is called on a deposit account with only one customer, we'll return an array with one customer. Do you agree that that's the correct approach to use?